.drone.starlark

```diff
@@ -520,15 +436,15 @@ config = {
 				'7.1',
 			],
 			'runCoreTests': True,
-			'cron': 'nightly',
+			'runAllSuites': True,
+			'numberOfParts': 3,
+			'emailNeeded': True,
+			# 'cron': 'nightly',
 		},
 		'core-webui-acceptance': {
-			'suites': [
-				'webUICore1',
-				'webUICore2',
-				'webUICore3',
-				'webUICore4',
-			],
+			'suites': {
+				'webUIall': 'core-webUI',
```

Comment:
```suggestion
				'webUIAll': 'core-webUI',
```

Reply:
@kiranparajuli589 Dipak is using the camelcase naming convention, so it's fine.